### ISSUE TITLE
Nice to have:  A link at the end of each exercise to take you to the next one

### ISSUE THREAD

**Issue by jung-thomas** [SAP-samples/cloud-btp-cli-api-codejam#26]

**Comment by qmacro** [comment]
Not having a link to the next exercise is deliberate, and has been part of the CodeJam-as-GitHub repo since the start (at least in the repos that I've created!). The flow of the CodeJam is such that all participants progress in parallel, and a discussion is had at the end of each exercise, with no-one jumping ahead to the next one until everyone's ready (see the [Exercises section of the main README](https://github.com/SAP-samples/cloud-btp-cli-api-codejam#exercises) for more context). Having links to the next exercise would put temptation there, which may be hard to resist for some :-)
